### ISSUE TITLE
data-srcset instead of srcset

### ISSUE THREAD

**Issue by meneerprins** [imgix/imgix-rails#60]
Would it be possible to generate data-src & data-srcset etc. instead of the normal attributes? So it's easier to make it compatible with lazyloading libraries.

**Comment by octave** [comment]
👍 

Yes like stated in this blog post: https://blog.imgix.com/2016/05/02/imgix-lazysizes

**Comment by piya23300** [comment]
I need it too.

I use React lib https://github.com/imgix/react-imgix#lazy-loading
There is 
```
attributeConfig={{
    src: "data-src",
    srcSet: "data-srcset",
    sizes: "data-sizes"
  }}
```

I think Rails lib should have it too.

**Comment by sherwinski** [comment]
Hey all,

We definitely agree that this gem should support lazy-loading! Unfortunately, we won't have the resources to revisit this until further down the road. For anyone searching for a solution in the meantime, I would encourage you to check out [native lazy-loading](https://caniuse.com/#feat=loading-lazy-attr) with `loading="lazy"`.

**Comment by lunks** [comment]
Support has been added on #108

**Comment by sherwinski** [comment]
Resolved in #108
